blocks/product-comparison-table/product-comparison-table.js

```diff
@@ -0,0 +1,101 @@
+import getMockData from './product-mock-data.js';
+
+const fakeData = getMockData();
+
+function buildPriceContainer(productName, numberOfDevices, elementToReplace) {
+  const priceContainer = document.createElement('div');
+  priceContainer.classList.add('product-comparison-price');
+
+  const productData = fakeData
+    .filter((product) => product.data.product.product_name === productName);
+  if (productData.length === 0) {
+    return;
+  }
+  const variationPrice = productData[0].data.product.variations[numberOfDevices][1];
+  const productVariationPrice = variationPrice.price;
+  const productVariationDiscountPrice = variationPrice.discount.discounted_price;
+  const priceLabel = variationPrice.currency_label;
+
+  if (productVariationDiscountPrice) {
+    const oldPriceContainer = document.createElement('div');
+    oldPriceContainer.classList.add('old-price-container');
+    oldPriceContainer.innerHTML = `<p>Old Price <span class='price-strike-through'>${productVariationPrice} ${priceLabel}</span></p>`;
```

Comment:
A `<del>` element could be used instead of the `<span + class>`

Reply:
updated